README.org

```diff
@@ -1009,6 +1009,9 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[http://endlessparentheses.com/][Endless Parentheses]] - Concise ramblings on Emacs productivity.
    - [[http://whattheemacsd.com/][What the .emacs.d!?]] - Sharing Emacs snippets and pastes with others.
    - [[https://hackr.io/tutorials/learn-emacs][Community Curated Emacs Resources]] - Top Recommended Resources.
+   - [[https://github.com/p3r7/awesome-elisp][awesome-elisp]] - An awesome list for Elisp development.
```

Comment:
Saw your comment @clemera on https://www.reddit.com/r/emacs/comments/jdcqc9/awesome_elisp_a_list_of_resources_about_emacs_lisp/ and https://github.com/p3r7/awesome-elisp/issues/6. 

"It would be great if these two projects could somehow join forces as they have a similar purpose. The devbook has lots of additional code mixed with links to resources. awesome-elisp seems to focus a bit more on a hierarchical TOC and then link to relevant packages. /u/eigenbahn has opened an issue to create an organization, maybe we could try to organize collaboration there."

Am thinking "let the dust settle" and wait a couple weeks on merging awesome-elisp. Would like to link to its final location, and one or two weeks of more active development and organization can't hurt.

Also contemplating that awesome-elisp and emacs-package-dev-handbook, could also potentially go under https://github.com/emacs-tw/awesome-emacs#emacs-lisp rather than here. Thoughts @kuanyui?

Reply:
Makes sense, I will comment here again once I think it's more plausible to merge.

Reply:
👍

Reply:
Also pinging @p3r7 here as he is the owner of the project and might be interested getting notified about this.

Reply:
Hello

> Also contemplating that awesome-elisp and emacs-package-dev-handbook, could also potentially go under [#emacs-lisp](https://github.com/emacs-tw/awesome-emacs#emacs-lisp) rather than here.

Yes, as both are very Elisp-centric (as compared to the more generic other resources) this seems more on point.

Still, it would break the logic of your document: mostly packages listed with no preamble to each section.

I can't really tell what's the best choice.

> Am thinking "let the dust settle" and wait a couple weeks on merging awesome-elisp. Would like to link to its final location, and one or two weeks of more active development and organization can't hurt.

I'm in line with you. The project is still very new and don't have that much traction yet (though hard to quantify).

We'll ideally need for more feedback regarding its structure and content.